### PR TITLE
Make experimental features public

### PR DESCRIPTION
**Pull request by mattwthompson**
### Description

Provide a brief description of the PR's purpose here.

### Checklist

- [ ] Add tests
- [ ] Lint
- [ ] Update docstrings

### CODE REVIEW

**Comment by mattwthompson** [comment]
Renders okay: https://openff-docs--703.org.readthedocs.build/projects/interchange/en/703/using/experimental.html

File: docs/using/experimental.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+or Jupyter notebook or equivalent
+
+```shell
+$ INTERCHANGE_EXPERIMENTAL=1 jupyter-notbook my_notebook.ipynb
+...
+```
+
+Feedback of any sort is useful - even (and especially) if something crashes early or produces surprising results. Please raise an [issue on GitHub](https://github.com/openforcefield/openff-interchange/issues/new) or email <info@openforcefield.org> sharing what you are trying to do and what happened instead.
```
Comment (mattwthompson) [inline docs/using/experimental.md:31]: Feel free to suggest a different email (`support@` ?) or none at all. Receiving support emails is not my favorite means of communication but I understand some people prefer it and/or might not be able to post publicly.

Comment (j-wags) [reply docs/using/experimental.md:31]: I wouldn't list an email at all.

File: docs/using/experimental.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+```shell
+$ INTERCHANGE_EXPERIMENTAL=1 python my_script.py
+...
+```
+
+or Jupyter notebook or equivalent
+
+```shell
+$ INTERCHANGE_EXPERIMENTAL=1 jupyter-notbook my_notebook.ipynb
```
Comment (j-wags) [inline docs/using/experimental.md:27]: Will this bork colab/binder?

**Review by j-wags — COMMENTED** [review]
I'm broadly in support of this, but don't have time for a full review.

**Pull request merged**